### PR TITLE
feat: add DCL (Database fundamentals page)

### PR DESCRIPTION
**Pull request by P13LIAM**
SQL
- [x] DDL (Data Definition Language)
- [ ] DQL (Data Query Language)
- [ ] DML (Data Manipulation Language)
- [x] DCL (Data Control Language)
- [ ] TCL (Transaction Control Language)

Demo:
image

image

### CODE REVIEW

File: docs/docs/05-database-fundamentals.md [hunk 1]
```diff
@@ -71,3 +71,23 @@ RENAME COLUMN old_name TO new_name;
 ```sql
 TRUNCATE TABLE table_name;
 ```
+
+### [1.4] DCL (Data Control Language)
+เป็นกลุ่มคำสั่งที่จะช่วยให้ผู้บริหารฐานข้อมูล (DBA)
+สามารถควบคุมฐานข้อมูลเพื่อ<u>กำหนดสิทธิการอนุญาต</u> (Grant)  หรือ <u>การยกเลิกการเข้าใช้</u> (Revoke) ฐานข้อมูล ซึ่งเป็นกระบวนการป้องกันความปลอดภัยในฐานข้อมูล
```
Comment (zkan) [inline docs/docs/05-database-fundamentals.md:77]: Not sure why Markdown doesn't render the underlined text. I guess we'd better use the normal text instead? 

```suggestion
สามารถควบคุมฐานข้อมูลเพื่อกำหนดสิทธิการอนุญาต (Grant)  หรือ การยกเลิกการเข้าใช้ (Revoke) ฐานข้อมูล ซึ่งเป็นกระบวนการป้องกันความปลอดภัยในฐานข้อมูล
```

Comment (zkan) [inline docs/docs/05-database-fundamentals.md]: Not sure why Markdown doesn't render the underlined text. I guess we'd better use the normal text instead? 

```suggestion
สามารถควบคุมฐานข้อมูลเพื่อกำหนดสิทธิการอนุญาต (Grant)  หรือ การยกเลิกการเข้าใช้ (Revoke) ฐานข้อมูล ซึ่งเป็นกระบวนการป้องกันความปลอดภัยในฐานข้อมูล
```

File: docs/docs/05-database-fundamentals.md [hunk 2]
```diff
@@ -71,3 +71,23 @@ RENAME COLUMN old_name TO new_name;
...
+
+* REVOKE: คำสั่งนี้จะ<u>ถอนสิทธิ์</u>การเข้าถึงของ users
+
+```sql
+REVOKE privileges_names ON object FROM user;
+```
+
+privileges_names ที่ GRANT หรือ REVOKE ได้
+
+!["fig2"](https://media.geeksforgeeks.org/wp-content/uploads/sqltable.jpg)
```
Comment (zkan) [inline docs/docs/05-database-fundamentals.md]: I think we should create a table in Markdown. This way we don't have to worry about the copyright. You can try [Markdown Tables generator](https://www.tablesgenerator.com/markdown_tables).

File: docs/docs/05-database-fundamentals.md [hunk 3]
```diff
@@ -71,3 +71,23 @@ RENAME COLUMN old_name TO new_name;
 ```sql
 TRUNCATE TABLE table_name;
 ```
+
+### [1.4] DCL (Data Control Language)
+เป็นกลุ่มคำสั่งที่จะช่วยให้ผู้บริหารฐานข้อมูล (DBA)
+สามารถควบคุมฐานข้อมูลเพื่อ<u>กำหนดสิทธิการอนุญาต</u> (Grant)  หรือ <u>การยกเลิกการเข้าใช้</u> (Revoke) ฐานข้อมูล ซึ่งเป็นกระบวนการป้องกันความปลอดภัยในฐานข้อมูล
+
+* GRANT: คำสั่งนี้<u>ให้สิทธิ์</u> users ในการเข้าถึงฐานข้อมูล
```
Comment (zkan) [inline docs/docs/05-database-fundamentals.md]:
```suggestion
* GRANT: คำสั่งนี้ให้สิทธิ์ผู้ใช้งานในการเข้าถึงฐานข้อมูล
```

File: docs/docs/05-database-fundamentals.md [hunk 4]
```diff
@@ -71,3 +71,23 @@ RENAME COLUMN old_name TO new_name;
...
+
+```sql
+GRANT privileges_names ON object TO user;
+```
+
+* REVOKE: คำสั่งนี้จะ<u>ถอนสิทธิ์</u>การเข้าถึงของ users
```
Comment (zkan) [inline docs/docs/05-database-fundamentals.md]:
```suggestion
* REVOKE: คำสั่งนี้จะถอนสิทธิ์การเข้าถึงของผู้ใช้งาน
```

File: docs/docs/05-database-fundamentals.md [hunk 5]
```diff
@@ -71,3 +71,23 @@ RENAME COLUMN old_name TO new_name;
...
+
+```sql
+REVOKE privileges_names ON object FROM user;
+```
+
+privileges_names ที่ GRANT หรือ REVOKE ได้
```
Comment (zkan) [inline docs/docs/05-database-fundamentals.md:93]: We can add more description here. For example:

```suggestion
โดย `privileges_names` คือชื่อ privilege ที่เราต้องการ `GRANT` หรือ `REVOKE` ส่วน `object` คือชื่อสิ่งที่เราต้องการจะปรับเปลี่ยนสิทธิ์ของผู้ใช้งานนั้น ๆ เช่น ชื่อตาราง และส่วน `user` คือ ชื่อผู้ใช้งาน
```

Anyway, you can come up with your own wording. 😃

**Review by zkan — CHANGES_REQUESTED** [review]

File: docs/docs/05-database-fundamentals.md (hunk 1 above)
Comment (kahnwong) [reply docs/docs/05-database-fundamentals.md]: normally I see people use **bold** or _italic_ to highlight words. I think we could do the same here

```
# markdown syntax

*bold* - bold text
_italic_ - italic text
```

Comment (P13LIAM) [reply docs/docs/05-database-fundamentals.md]: Got it, Thank you
fixed

File: docs/docs/05-database-fundamentals.md (hunk 2 above)
Comment (P13LIAM) [reply docs/docs/05-database-fundamentals.md]: Great tool, Thank you

File: docs/docs/05-database-fundamentals.md (hunk 3 above)
Comment (P13LIAM) [reply docs/docs/05-database-fundamentals.md]: fixed

File: docs/docs/05-database-fundamentals.md (hunk 4 above)
Comment (P13LIAM) [reply docs/docs/05-database-fundamentals.md]: fixed

File: docs/docs/05-database-fundamentals.md (hunk 5 above)
Comment (P13LIAM) [reply docs/docs/05-database-fundamentals.md:93]: done

**Review by zkan — APPROVED** [review]
Looking great! @petongz

**Pull request merged**